browser/mapping.go

```diff
@@ -280,8 +280,15 @@ func mapElementHandle(vu moduleVU, eh *common.ElementHandle) mapping {
 			return mapFrame(vu, f), nil
 		},
 		"press": eh.Press,
-		"screenshot": func(opts goja.Value) goja.ArrayBuffer {
-			return eh.Screenshot(opts, vu.LocalFilePersister)
+		"screenshot": func(opts goja.Value) ([]byte, error) {
+			ctx := vu.Context()
+
+			popts := common.NewElementHandleScreenshotOptions(eh.Timeout())
+			if err := popts.Parse(ctx, opts); err != nil {
+				return nil, fmt.Errorf("parsing frame screenshot options: %w", err)
+			}
+
+			return eh.Screenshot(popts, vu.LocalFilePersister) //nolint:wrapcheck
```

Comment:
Should we return an `ArrayBuffer` here? I remember that transferring a large number of bytes can be problematic. WDYT @mstoykov?

Reply:
Right, good spot. I forgot to make a note here to highlight the change. I'm going to run some tests to see how this may have affect the buffer that is returned.

Reply:
I ran some code in PW and k6 with screenshot and they were printing different results (PW was printing unreadable values such as `u��I���*` and k6 browser was printing numbers such as `9,148,148,164,`). I'm not entirely sure how to work with the buffer either. I think it's safer to change it to work with `ArrayBuffer` and keep the existing behaviour for now and then come back to fully grasping the buffer at a later stage 👍

Reply:
Resolved in 59ce5ab508a2cf0ce3d40feb71e0152aeb3c898f and 0a093fdb5ca8f8e9398505f59172153bfb4ee5c1

Reply:
I don't think I have enough information to answer this question.

Is this still a problem or have you fixed it in the following PRs.

And if it is (or you want me to confirm) can you please explain more what is happening.

Otherwise - yes if you are working with binary data you should return ArrayBuffer or some TypedArray( for example Uint8Array) which usually is more involved.

Reply:
From what I understand, we should continue to use `ArrayBuffer` since we're working with binary data.

Reply:
Yep, it was a binary of screenshots, so we're working with `ArrayBuffer` (again) 👍